openfisca_france/model/revenus/remplacement/indemnites_journalieres_securite_sociale.py

```diff
@@ -74,3 +73,9 @@ def function(self, simulation, period):
         )
 
         return period, result
+
+class date_arret_de_travail(Variable):
+    column = DateCol(default = nan)
```

Comment:
In `rsa_indemnites_journalieres_activite` I see:

```
is_date_arret_de_travail_undefined = (date_arret_de_travail == datetime64('NaT'))
```

so perhaps the JSON serialization and parsing has to be updated.

Or use a "+infinity" date (ex: 2999-12-31) to be sure that by default the person is never in "arrêt de travail" situation.